composer.json

```diff
@@ -14,6 +14,7 @@
         "symfony/orm-pack": "^1.0",
         "symfony/security-bundle": "5.0.*",
         "symfony/serializer-pack": "^1.0",
+        "symfony/twig-pack": "^1.0",
```

Comment:
Der Robin hatte gerade sich die Mühe gemacht die Dependencies des "packs" direkt in die composer.json einzubinden

Reply:
damit war twig in prod aber nicht vorhanden :-(